examples/google-alloydb-vectorstore-example/alloydb_vectorstore_example.go

```diff
@@ -80,35 +86,24 @@ func main() {
 	}
 
 	// Initialize table for the Vectorstore to use. You only need to do this the first time you use this table.
-	vectorstoreTableoptions, err := &alloydbutil.VectorstoreTableOptions{
-		TableName:  "table",
-		VectorSize: 768,
-	}
-
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	err = pgEngine.InitVectorstoreTable(ctx, *vectorstoreTableoptions,
-		[]alloydbutil.Column{
-			alloydbutil.Column{
-				Name:     "area",
-				DataType: "int",
-				Nullable: false,
-			},
-			alloydbutil.Column{
-				Name:     "population",
-				DataType: "int",
-				Nullable: false,
-			},
+	vectorstoreTableoptions := &alloydbutil.VectorstoreTableOptions{
+		TableName:          "testtable",
+		VectorSize:         768,
+		MetadataJsonColumn: "langchain_metadata",
```

Comment:
metadatajsoncolmn should default to this value; therefore we shouldnt set it here. Is that correct? Same with the id column